RNTester/Podfile

```diff
@@ -1,10 +1,13 @@
 source 'https://cdn.cocoapods.org/'
 
-platform :ios, '9.0'
-
 require_relative '../scripts/autolink-ios'
 
+# Otherwise duplicate UUIDs are being generated between the iOS and macOS targets.
+# FIXME: https://github.com/microsoft/react-native/issues/210
+install! 'cocoapods', :deterministic_uuids => false
+
 target 'RNTester' do
+  platform :ios, '9.0'
```

Comment:
We should tag our diffs between what we have and what facebook has so if there's ever a merge conflict, we know that our change was intentional and can fairly quickly fix the problem. If diffs are untagged, people generally take what fb's master has, which could regress osx support. I'd just use the same tag everywhere, like "Required for OSX Support" or something along those lines. Could add a GitHub bug number to it as well

Reply:
Done